CONTRIBUTING.md

```diff
@@ -1278,55 +1278,37 @@ our [CI wiki](https://github.com/pytorch/pytorch/wiki/Debugging-using-with-ssh-f
 ### Which commit is used in CI?
 
 For CI run on `master`, this repository is checked out for a given `master`
-commit, and CI is run on that commit (there isn't really any other choice). For
-PRs, however, it's a bit more complicated. Consider this commit graph, where
+commit, and CI is run on that commit (there isn't really any other choice).
+
+For PRs, however, it's a bit more complicated. Consider this commit graph, where
 `master` is at commit `A`, and the branch for PR #42 (just a placeholder) is at
 commit `B`:
 
 ```
-       o---o---B (refs/pull/42/head)
+       o---o---B (pr-commit)
       /         \
-     /           C (refs/pull/42/merge)
+     /           C (merged-commit)
     /           /
----o---o---o---A (refs/heads/master)
+---o---o---o---A (merge-destination) - usually master
```

Comment:
Do you know if `refs/pull/42/base` or `refs/pull/42/target`exists?

Reply:
Running `git ls-remote` shows that something similar to `refs/pull/42/base` does exist, but it doesn't show any `refs/pull/42/target`

Thought that may be cuz the target is usually `master`